<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowBase</class>
 <widget class="QMainWindow" name="MainWindowBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetImages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Images to process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Config</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditConfig">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>40</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string notr="true"/>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="plainText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoad">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionGo"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuConfig"/>
   <addaction name="menuConnect"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="text">
    <string>Go</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>addFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>clearFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonLoad</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>loadConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>addFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>loadConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>clearFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonGo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>go()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBase</receiver>
   <slot>go()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addFiles()</slot>
  <slot>clearFiles()</slot>
  <slot>loadConfig()</slot>
  <slot>saveConfig()</slot>
  <slot>go()</slot>
 </slots>
</ui>
